### PR TITLE
feat(mvp): implement client db connector

### PR DESCRIPTION
**Pull request by ulya-sidorina**

### CODE REVIEW

File: internal/client-db-connector/connector.go [hunk 1]
```diff
@@ -1,4 +1,177 @@
...
+}
+
+type ClientDbConnectorImpl struct {
+}
+
+func (d *ClientDbConnectorImpl) ExportToS3(ctx context.Context, settings types.ExportToS3Settings) (string, error) {
+	clientDbConnectionString := settings.ClientDb.Endpoint + settings.ClientDb.DatabaseName
+
+	xlog.Info(ctx, "Connecting to client db", zap.String("dsn", clientDbConnectionString))
+	db, connErr := ydb.Open(ctx, clientDbConnectionString, ydb.WithAnonymousCredentials())
```
Comment (robdrynkin) [inline internal/client-db-connector/connector.go:34]: we don't want to create ydb driver for each export, move it to the ClientDbConnectorImpl

File: internal/client-db-connector/connector.go [hunk 2]
```diff
@@ -1,4 +1,177 @@
...
+	)
+
+	response, importErr := importClient.ImportFromS3(
+		ctx,
+		&Ydb_Import.ImportFromS3Request{
+			OperationParams: &Ydb_Operations.OperationParams{
+				OperationTimeout: durationpb.New(time.Second),
+				CancelAfter:      durationpb.New(time.Second),
+			},
+			Settings: &Ydb_Import.ImportFromS3Settings{
```
Comment (robdrynkin) [inline internal/client-db-connector/connector.go:113]: Why do we need types.ImportFromS3Settings if its almost the same as Ydb_Import.ImportFromS3Settings ?

File: internal/client-db-connector/connector_test.go
Comment (robdrynkin) [inline internal/client-db-connector/connector_test.go:1]: connector_test.go -> connector_mock.go ?

File: internal/client-db-connector/connector.go
Comment (robdrynkin) [inline internal/client-db-connector/connector.go:1]: replace "-" with "_" in folder name

File: internal/client-db-connector/connector_test.go
Comment (bma13) [reply internal/client-db-connector/connector_test.go:1]: >file a name ending in "_test.go". The file will be excluded from regular package builds but will be included when the "go test" command is run. 
https://pkg.go.dev/testing

Storing mocks in _test.go makes sense

**Review by bma13 — APPROVED** [review]

**Pull request merged**